.github/workflows/ci.yml

```diff
@@ -126,6 +131,9 @@ jobs:
         SPARSE_BACKEND: ${{ matrix.backend }}
       run: |
         cd ${GITHUB_WORKSPACE}/array-api-tests
+        if [ "${SPARSE_BACKEND}" = "Finch" ]; then
+            python -c 'import finch'
+        fi
```

Comment:
Why do we need these lines?

Reply:
thats the parallel julia install problem: https://github.com/JuliaPy/pyjuliapkg/issues/19